Licensing: update topic for new license mechanism (#8772)
DevExpress/devextreme-documentation: concepts/Common/Licensing/10 Validate Your License Key/10 Manual Configuration.md

> After you obtain your DevExpress license key and register it on your machine, use the `devextreme-license` CLI tool to generate a DevExtreme-compatible runtime key and register it in your application. Configuration differs based on whether your project is *modular* (files added via `import`/`require` statements) or *non-modular* (files referenced via `<script>` tags):

Reviewer: ```suggestion
After you obtain your DevExpress license key and register it on your machine, use the `devextreme-license` CLI tool to generate a DevExtreme-compatible runtime key and register it in your application. The configuration differs depending on whether your project is modular (files added with `import`or `require`) or non-modular (files referenced with `<script>` tags):
```

> After you obtain your DevExpress license key, use the `devextreme-license` CLI tool to generate a DevExtreme-compatible runtime key and register it in your application. Configuration differs based on whether your project is *modular* (files added via `import`/`require` statements) or *non-modular* (files referenced via `<script>` tags).

Reviewer 2: After you obtain your DevExpress license key, ... and register it on your machine

Reviewer 2: Since we are talking here about two main approaches we can provide the exact link for each of them
[modular](https://docs.devexpress.com/GeneralInformation/405494/trial-register/set-up-your-dev-express-license-key?v=26.1#option-1-modular-apps-angular-react-vue)
[Non-Modular](https://docs.devexpress.com/GeneralInformation/405494/trial-register/set-up-your-dev-express-license-key?v=26.1#option-2-non-modular-applications-jquery-cdn-based)
